examples/xml-from-record-conversion/xml_from_record_conversion.md

```diff
@@ -0,0 +1,11 @@
+# XML from record conversion
```

Comment:
```suggestion
# XML from record conversion
```
This title is confusing. Should describe from what to what is the conversion.

Reply:
I have changed the title to `Conversion of Ballerina record to XML`.

Reply:
This name should consistent with the XML-JSON conversion

Reply:
Let's name the BBEs like,

XML to JSON conversion
JSON to XML conversion
XML to Record conversion
Record to XML conversion